.github/workflows/python-app.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Escape Roomba
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
```

Comment:
(the blank lines around "runs-on:" look slightly odd to my eye, but totally nbd)

Reply:
Fixed.